packages/lesswrong/components/review/ReviewVotingPage.tsx

```diff
@@ -20,15 +20,16 @@ import { forumTypeSetting } from '../../lib/instanceSettings';
 import { annualReviewStart } from '../../lib/publicSettings';
 
 export const REVIEW_YEAR = 2020
-const VOTING_VIEW = "roughVoting2020" // unfortunately this can't just inhereit from REVIEW_YEAR. It needs to exactly match a view-type so that the type-check of the view can pass.
+const VOTING_VIEW = "reviewVoting" // unfortunately this can't just inhereit from REVIEW_YEAR. It needs to exactly match a view-type so that the type-check of the view can pass.
 const REVIEW_COMMENTS_VIEW = "reviews2020"
 const userVotesAreQuadraticField: keyof DbUser = "reviewVotesQuadratic2020";
 const isEAForum = forumTypeSetting.get() === "EAForum"
+const isLWForum = forumTypeSetting.get() === "LessWrong"
 
 export const currentUserCanVote = (currentUser: UsersCurrent|null) => {
   if (!currentUser) return false
-  if (!isEAForum && new Date(currentUser.createdAt) < new Date(`${REVIEW_YEAR}-01-01`)) return false
-  if (isEAForum && new Date(currentUser.createdAt) < new Date(annualReviewStart.get())) return false
+  if (isLWForum && new Date(currentUser.createdAt) > new Date(`${REVIEW_YEAR}-01-01`)) return false
+  if (isEAForum && new Date(currentUser.createdAt) > new Date(annualReviewStart.get())) return false
```

Comment:
Wait, seriously? You gotta be kidding. Those comparisons have to be the wrong way around. I didn't test it but I was like (internal feeling of) 98% confident. I gotta try. Very apologies if I was wrong.

Reply:
Nope, I was right.

Reply:
Asdf looking at the wrong component.

Reply:
I think I had what the function did turned around in my head. :\

Reply:
Yep, except my pride